src/common/Utility.js

```diff
@@ -218,50 +218,63 @@
         return Object.prototype.toString.call(obj) === "[object Array]";
     }
 
-    //  Template   ---
-    //  https://github.com/Matt-Esch/string-template (MIT)
-    var nargs = /\{([0-9a-zA-Z_\s\[\]]+)\}/g;
-    function template(string) {
-        var args;
-
-        if (arguments.length === 2 && typeof arguments[1] === "object") {
-            args = arguments[1];
-        } else {
-            args = new Array(arguments.length - 1);
-            for (var i = 1; i < arguments.length; ++i) {
-                args[i - 1] = arguments[i];
-            }
-        }
+    function removeHTMLFromString(str, div){
+        div = div ? div : document.createElement("div");
+        div.innerHTML = str;
+        return div.textContent || div.innerText || "";
+    }
 
-        if (!args || !args.hasOwnProperty) {
-            args = {};
+//  Template   ---
+//  https://github.com/Matt-Esch/string-template (MIT)
+var nargs = /\{([0-9a-zA-Z_\s\[\]]+)\}/g;
+var nargs2 = /\(([0-9a-zA-Z_\s\[\]]+)\)/g;
```

Comment:
How about:  `/\{\{([0-9a-zA-Z_\s\[\]]+)\}\}/g;`
So if {a} then its verbatim, but if its:  {{a}} then it strips the HTML?

Reply:
That sounds good to me. I'll test it out